Framework/Crystal/src/FindSXPeaks.cpp

```diff
@@ -152,26 +206,27 @@ void FindSXPeaks::exec() {
     double rightBinEdge = *std::next(leftBinPosition);
     double tof = 0.5 * (leftBinEdge + rightBinEdge);
 
-    // If no detector found, skip onto the next spectrum
-    if (!spectrumInfo.hasDetectors(static_cast<size_t>(i))) {
-      continue;
-    }
-    if (!spectrumInfo.hasUniqueDetector(i)) {
-      std::ostringstream sout;
-      sout << "Spectrum at workspace index " << i
-           << " has unsupported number of detectors.";
-      throw std::runtime_error(sout.str());
+    double phi = -999;
```

Comment:
It was for testing. Good catch - I think setting it to `inf` would make a good canary value as not many calculations result in infinity (vs `NaN`)